src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -34,11 +35,13 @@ class ErrorBoundary extends React.Component<Props, State> {
     }
 
     render() {
-        if (this.state.error) {
-            return this.props.component;
+        const { errorComponent: ErrorComponent, children } = this.props;
```

Comment:
Can you also add the props onError check above

Reply:
The defaultProp sets it to noop, so should be good